scripts/packages/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -2,22 +2,18 @@ $ErrorActionPreference = 'Stop'; # stop on all errors
 $packageName = 'bazel'
 
 $toolsDir = Split-Path -parent $MyInvocation.MyCommand.Definition
-$json = gc "$toolsDir\params.json"
-$p = (($json) -join "`n") | convertfrom-json
+$paramsText = get-content "$($toolsDir)\params.txt"
+$p = $paramsText.Split("`n")
 
 $packageDir = Split-Path -parent $toolsDir
-$binRoot = (Get-ToolsLocation) -replace "\\", "/"
 
-write-host "Read params from json"
-write-host (convertto-json $p)
+write-host "Read params from text file"
+write-host $p
 
 Install-ChocolateyZipPackage -PackageName "$packageName" `
-  -Url "$($p.package.uri)" `
```

Comment:
By the way - these are removed because there's no 32-bit build. These not being present causes chocolatey to print a friendly error to a 32-bit user saying it's not possible to install this there.